### PR TITLE
SHRUG!

### PR DESCRIPTION
**Pull request by krzyzacy**

### CODE REVIEW

**Comment by krzyzacy** [comment]
/assign @spxtr @BenTheElder
:-(

**Comment by BenTheElder** [comment]
/lgtm
¯\\\_(ツ)\_/¯

**Comment by krzyzacy** [comment]
that should match, I'll bump hook
```
{
    "id": 521200414,
    "url": "https://api.github.com/repos/kubernetes/test-infra/labels/%C2%AF%5C_(%E3%83%84)_/%C2%AF",
    "name": "¯\\_(ツ)_/¯",
    "color": "f9d0c4",
    "default": false
  }
```

**Comment by fejta** [comment]
This is a superfluous PR!
/approve

File: prow/plugins/shrug/shrug.go [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+	"k8s.io/test-infra/prow/github"
+	"k8s.io/test-infra/prow/plugins"
+)
+
+const pluginName = "shrug"
+
+var (
+	shrugLabel = "¯\\_(ツ)_/¯"
+	shrugRe    = regexp.MustCompile(`(?mi)^/shrug\s*$`)
```
Comment (fejta) [inline prow/plugins/shrug/shrug.go]: I should be able to remove the shrug somehow! :D

Comment (krzyzacy) [reply prow/plugins/shrug/shrug.go]: okay, what's the English word opposite to shrug?

Comment (kargakis) [reply prow/plugins/shrug/shrug.go]: how about "/remove-shrug" or "/shrug cancel" or "/shrug remove"?

Comment (fejta) [reply prow/plugins/shrug/shrug.go]: ¯\\_(ツ)_/¯

Comment (kargakis) [reply prow/plugins/shrug/shrug.go]: _/(ツ)\_

Comment (BenTheElder) [reply prow/plugins/shrug/shrug.go]: `/shrug cancel` seems appropriate, but only if the bot replies with something like:

Ok _ /¯(ツ)¯\ _

:smile:

**Pull request merged**